### PR TITLE
Change Upload Success Link

### PR DESCRIPTION
**Pull request by AnonymousWalker**
### Issue #16 

Now the prompted link after uploaded will be: **_read.bibletranslationtools.org..._**
e.g. `http://read.bibletranslationtools.org/u/clash/bn_mat_text_ulb/`

instead of 

image

### CODE REVIEW

File: src/config/user-setting.json [hunk 1]
```diff
@@ -164,6 +164,15 @@
         "enabled": true,
         "visible": true
     },
+    {
+        "name": "readerserver",
+        "mainText": "Reader Server",
+        "value": "http://read.bibletranslationtools.org/u",
+        "default": "http://read.bibletranslationtools.org/u",
```
Comment (AnonymousWalker) [inline src/config/user-setting.json:171]: e.g.
`http://read.bibletranslationtools.org/u/clash/bn_mat_text_ulb`

**Comment by PurpleGuitar** [comment]
Nice job on this so far!  Your changes to the settings are working well.

However, there is another requirement that I forgot to mention.  :)  This setting should automatically change to the corresponding URL on DCS when the user changes the Server Suite setting from WACS to DCS.  The corresponding URL on DCS is: https://door43.org/u/<user_id>/<repo_id>

To do this:

- Add a default "readerserver" attribute to lines 142 and 143 of src/config/user-setting.json.
- Set the value on change around line 298 of src/elements/ts-settings/ts-settings.html

Let me know if you have questions.  Keep up the good work!

File: src/elements/ts-settings/ts-settings.html [hunk 2]
```diff
@@ -296,6 +296,7 @@
             if (name === "serversuite") {
                 App.configurator.setUserSetting("dataserver", value.dataserver);
                 App.configurator.setUserSetting("mediaserver", value.mediaserver);
+                App.configurator.setUserSetting("readerserver", value.mediaserver);
```
Comment (PurpleGuitar) [inline src/elements/ts-settings/ts-settings.html:299]: This should be `value.readerserver`, not `value.mediaserver`.

**Review by PurpleGuitar — CHANGES_REQUESTED** [review]

File: src/elements/ts-settings/ts-settings.html (hunk 2 above)
Comment (AnonymousWalker) [reply src/elements/ts-settings/ts-settings.html]: great catch! I pushed the correct one.

**Review by PurpleGuitar — APPROVED** [review]
Looks good!  Nice work.

**Pull request merged**